src/nedap/utils/spec/impl/check.cljc

```diff
@@ -1,59 +1,66 @@
 (ns nedap.utils.spec.impl.check
   (:require
+   #?(:clj [clojure.stacktrace :refer [print-stack-trace]])
    #?(:clj [clojure.spec.alpha :as spec] :cljs [cljs.spec.alpha :as spec])
-   #?(:clj [clojure.stacktrace])
+   [clojure.string :as str]
    [expound.alpha :as expound])
   #?(:cljs (:require-macros [nedap.utils.spec.impl.check])))
 
-(defn print-stack-frames
-  "numbers of stack frames to print before raising the exception
+#?(:clj
+   (defn print-stack-frames
+     "numbers of stack frames to print before raising the exception"
+     []
+     (Long/parseLong (System/getProperty "nedap.utils.spec.print-stack-frames" "0"))))
+
+#?(:clj
+   (defn print-symbol
+     "Returns the fully qualified symbol to used to print a spec failure.
 
-  note: only available in clj"
-  []
-  #?(:cljs 0
-     :clj (Long/parseLong (System/getProperty "nedap.utils.spec.print-stack-frames" "0"))))
+     The reporter is to be used as a fallback if the exception may be lost."
+     []
+     (symbol (System/getProperty "nedap.utils.spec.print-symbol" "clojure.core/println"))))
```

Comment:
So if I don't want printing, I need to supply `identity`?

Reply:
You can supply clojure.core/comment too